conf/production/app_prod.conf

```diff
@@ -2,6 +2,7 @@ appname = docker-logs-agent
 httpport = 7001
 runmode = prod
 autorender = false
-copyrequestbody = true
+copyrequestbody = false
 EnableDocs = false
 docker_api_version = 1.39
+EnableGzip = true
```

Comment:
why are these config vars mixed case? (both staring with Uppercase and lowercase)